app/models/concerns/has_custom_fields.rb

```diff
@@ -160,24 +160,26 @@ def clear_custom_fields
     @custom_fields_orig = nil
   end
 
+  class N1NotPreLoadedError < StandardError; end
```

Comment:
I found the class name confusing here because N1 didn't make sense initially. Also, I'm wondering if we actually need to include details about N+1 queries problem in the error name. I think `NotPreloadedError` itself is fine since the error message will provide details about why the error is being raised.

Reply:
Sure, will change it, just want something other than `StandardError`